changelog/6962.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed bug where parametrized fixtures were not being cached correctly, being recreated every time.
```

Comment:
```suggestion
Parametrization parameters are now compared using `==` instead of `is` (`is` is still used as a fallback if the parameter does not support `==`).
This fixes use of parameters such as lists, which have a different `id` but compare equal, causing fixtures to be re-computed instead of being cached.
```